src/rescue_password_menu.c

```diff
@@ -12,14 +12,244 @@
+        0x00, 0x00, 0x00, 0x00,
+        0x03,
+        0x03, 0x00, 0x09, 0x00,
+        0x17, 0x05,
+        0x05, 0x00,
+        NULL
+
+};
+
+const u8 Error_80E7268[];
+const u8 IncorrectPassword_80E7250[];
+const u8 CheckAgain_80E7234[];
+
+const struct MenuItem gUnknown_80E7214[4] = {
+    {Error_80E7268, 0xC},
+    {IncorrectPassword_80E7250, 0xC},
+    {CheckAgain_80E7234, 0xC},
+    {NULL, 0xA},
+};
+
+ALIGNED(4) const u8 CheckAgain_80E7234[] = _("{CENTER_ALIGN}Please check it again.");
+ALIGNED(4) const u8 IncorrectPassword_80E7250[] = _("{CENTER_ALIGN}Incorrect password!");
+ALIGNED(4) const u8 Error_80E7268[] = _("{CENTER_ALIGN}{COLOR_1 RED}Error!{END_COLOR_TEXT_1} ");
+
+const struct UnkTextStruct2 gUnknown_80E7278 = {
+        0x00, 0x00, 0x00, 0x00,
+        0x03,
+        0x03, 0x00, 0x09, 0x00,
+        0x17, 0x05,
+        0x05, 0x00,
+        NULL
+
+};
+
+const u8 Success_80E72D8[];
+const u8 PasswordAccept_80E72B4[];
+const u8 DATA_80E72B0[];
+
+const struct MenuItem gUnknown_80E7290[4] = {
+    {Success_80E72D8, 0xC},
+    {PasswordAccept_80E72B4, 0xC},
+    {DATA_80E72B0, 0xC},
+    {NULL, 0xA},
+};
+
+ALIGNED(4) const u8 DATA_80E72B0[] = "  ";
+ALIGNED(4) const u8 PasswordAccept_80E72B4[] = _("{CENTER_ALIGN}The password has been accepted.");
+ALIGNED(4) const u8 Success_80E72D8[] = _("{CENTER_ALIGN}{COLOR_1 LIGHT_BLUE_2}Success!{END_COLOR_TEXT_1} ");
+
+const struct UnkTextStruct2 gUnknown_80E72EC = {
+        0x00, 0x00, 0x00, 0x00,
+        0x03,
+        0x03, 0x00, 0x09, 0x00,
+        0x17, 0x05,
+        0x05, 0x00,
+        NULL
+
+};
+
+const u8 NotSOSMail_80E7324[];
+
+const struct MenuItem gUnknown_80E7304[4] = {
+    {Error_80E7268, 0xC},
+    {NotSOSMail_80E7324, 0xC},
+    {CheckAgain_80E7234, 0xC},
+    {NULL, 0xA},
+};
+
+ALIGNED(4) const u8 NotSOSMail_80E7324[] = _("{CENTER_ALIGN}This is not an {COLOR_1 LIGHT_BLUE_2}SOS Mail{END_COLOR_TEXT_1}.");
+
+const struct UnkTextStruct2 gUnknown_80E7344 = {
+        0x00, 0x00, 0x00, 0x00,
+        0x03,
+        0x03, 0x00, 0x09, 0x00,
+        0x17, 0x05,
+        0x05, 0x00,
+        NULL
+
+};
+
+const u8 ReceiveSOSMail_80E737C[];
+
+const struct MenuItem gUnknown_80E735C[4] = {
+    {Error_80E7268, 0xC},
+    {ReceiveSOSMail_80E737C, 0xC},
+    {CheckAgain_80E7234, 0xC},
+    {NULL, 0xA},
+};
+
+ALIGNED(4) const u8 ReceiveSOSMail_80E737C[] = _("{CENTER_ALIGN}This {COLOR_1 LIGHT_BLUE_2}SOS Mail{END_COLOR_TEXT_1} has been received already.");
+
+const struct UnkTextStruct2 gUnknown_80E73AC = {
+        0x00, 0x00, 0x00, 0x00,
+        0x03,
+        0x03, 0x00, 0x09, 0x00,
+        0x17, 0x05,
+        0x05, 0x00,
+        NULL
+
+};
+
+const u8 sUnknown_80E73E4[];
+
+const struct MenuItem gUnknown_80E73C4[4] = {
+    {Error_80E7268, 0xC},
+    {sUnknown_80E73E4, 0xC},
+    {CheckAgain_80E7234, 0xC},
+    {NULL, 0xA},
+};
+
+ALIGNED(4) const u8 sUnknown_80E73E4[] = _("{CENTER_ALIGN}This is not an #CGA-OK Mail{END_COLOR_TEXT_1}.");
```

Comment:
The charmap macros were used up until this point in the file. Let's keep using them for the rest of the file.